.circleci/config.yml

```diff
@@ -100,6 +133,7 @@ workflows:
   version: 2
   build_test_and_deploy:
     jobs:
+      - build_nginx
```

Comment:
_question_ The other `build` requires "test". Should this require "test" too?
_question_ Should we rename this to `build-web`?

Reply:
Yeah, even though the test job runs Python unit tests and nothing to do with nginx or the generation of static assets, I agree that, because we have single deploy at the mo, it makes sense to make the `build_nginx` depend on the `test` job.

And agree on the renaming.